setup.py

```diff
@@ -133,13 +138,16 @@ def build_extension(self, ext):
 
 setup(
     name='pylibczi',
```

Comment:
This will fail because a `pylibczi` package already exists. Need to change this an rename any reference of `pylibczi` to `aicspylibczi`